spec/support/ar_classes.rb

```diff
@@ -1,7 +1,24 @@
-ActiveRecord::Base.establish_connection(
-  :adapter => 'sqlite3',
-  :database => ':memory:'
-)
+FileUtils.rm_f('./test.db')
+
+def establish_active_record_connection
+  ActiveRecord::Base.establish_connection(
+    :adapter => 'sqlite3',
+    :database => './test.db'
+  )
+end
+
+def clear_active_record_connection
+  ActiveRecord::Base.connection_handler.clear_all_connections!
+  ActiveRecord::Base.connection_handler.connection_pool_list.each do |pool|
+    if Rails.version.to_f >= 5.0
+      ActiveRecord::Base.connection_handler.remove_connection(pool.spec.name)
+    else
+      ActiveRecord::Base.connection_handler.remove_connection(ActiveRecord::Base)
+    end
+  end
+end
+
+establish_active_record_connection
```

Comment:
This should be removed, we don't want every other test having a database connection.

Reply:
Good point, but it's not a subject of the PR. Actually all tests require a database connection due to the `*Model` classes at `ar_classes.rb`. When disabling AR support on the specs overall, everything is broken. Disabling the database connection on all tests sounds like a bigger refactoring.

Reply:
> Actually all tests require a database connection due to the *Model classes at ar_classes.rb. When disabling AR support on the specs overall, everything is broken. 

Currently there is no database configured so I'm not sure thats true, if your changes cause things to fail your changes are at fault, applying a global config to the specs is not a valid fix.

Reply:
> Currently there is no database configured so I'm not sure thats true

https://github.com/rspec/rspec-rails/blob/master/spec/support/ar_classes.rb#L1-L19

Reply:
Hmm ok, well make it fail with that.

Reply:
We have `in_sub_process` helpers if you need to change something, but I don't want global configuration changes for just this spec, it shouldn't be necessary.